skfp/bases/base_substructure_fp.py

```diff
@@ -43,8 +42,10 @@ class BaseSubstructureFingerprint(BaseFingerprintTransformer):
         Number of inputs processed in each batch. ``None`` divides input data into
         equal-sized parts, as many as ``n_jobs``.
 
-    verbose : int, default=0
+    verbose : int or ``dict``, default=0
```

Comment:
You don't need backticks for parameter types, only for actual descriptions. This is one of many weird things in Sphinx and scikit-learn standard for it. Sorry for not being precise about it earlier.

Reply:
Done